### PR TITLE
Release announcement for version 0.6.11.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: _posts/2020-07-07-Solidity-0611-release-announcement.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  mapping(address => bool) public participants;
+
+  function register() public override {
+    participants[msg.sender] = true;
+    emit Registered(msg.sender);
+  }
+}
+```
+
+The derived contract ``MyGathering`` in the example code above will produce the following userdoc:
```
Comment (axic) [inline _posts/2020-07-07-Solidity-0611-release-announcement.md:54]: Are you sure? It did not include the event for the deposit contract for me.

File: _posts/2020-07-07-Solidity-0611-release-announcement.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+
+### Language Features:
+ * General: Add unit denomination ``gwei``
+ * Yul: Support ``linkersymbol`` builtin in standalone assembly mode to refer to library addresses.
+ * Yul: Support using string literals exceeding 32 bytes as literal arguments for builtins.
+
+
+### Compiler Features:
+ * NatSpec: Add fields "kind" and "version" to the JSON output.
```
Comment (axic) [inline _posts/2020-07-07-Solidity-0611-release-announcement.md:90]:
```suggestion
 * NatSpec: Add fields ``kind`` and ``version`` to the JSON output.
```

Should also change in the actual changelog.

File: _posts/2020-07-07-Solidity-0611-release-announcement.md (hunk 1 above)
Comment (chriseth) [reply _posts/2020-07-07-Solidity-0611-release-announcement.md:54]: Just recompiled and tried again: it does contain it.

File: _posts/2020-07-07-Solidity-0611-release-announcement.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+so stay tuned!
+
+
+Furhermore, NatSpec is now supported on events.
+
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.11;
+interface Gathering {
+  /// The address `participant` just registered for the gathering.
```
Comment (axic) [inline _posts/2020-07-07-Solidity-0611-release-announcement.md:37]: This becomes a notice without an explicit `@notice` tag?

Comment (hrkrshnn) [reply _posts/2020-07-07-Solidity-0611-release-announcement.md:37]: @axic Yes. Notice is the "default."

**Comment by chriseth** [comment]
Updated.

File: _posts/2020-07-07-Solidity-0611-release-announcement.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+ * Type Checker: Do not disallow assigning to calldata variables.
+ * Type Checker: Fix internal error related to ``using for`` applied to non-libraries.
+ * Wasm backend: Fix code generation for for-loops with pre statements.
+ * Wasm backend: Properly support both ``i32.drop`` and ``i64.drop``, and remove ``drop``.
+ * Yul: Disallow the same variable to occur multiple times on the left-hand side of an assignment.
+ * Yul: Fix source location of variable multi-assignment.
+
+A big thank you to all contributors who helped make this release possible!
+
+Download the new version of Solidity [here](https://github.com/ethereum/solidity/releases/tag/v0.6.10).
```
Comment (axic) [inline _posts/2020-07-07-Solidity-0611-release-announcement.md:111]: Wrong link.

**Pull request merged**